Bottom: b31c16549d2d5f34412f7266d7c3254eab3dd876
Top:    7965f26daa7271fff7e42c8fb3092fc9b022c632
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-13 00:22:07 +0100

Refresh of apply-the-default-hidnogui-hid

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index ea0a0dd..091f472 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2330,6 +2330,7 @@ hid_gtk_init ()
   ghid_hid.flush_debug_draw         = ghid_flush_debug_draw;
   ghid_hid.finish_debug_draw        = ghid_finish_debug_draw;
 
+  apply_default_hid (&ghid_hid, 0);
   hid_register_hid (&ghid_hid);
 #include "gtk_lists.h"
 }
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 4f7e12a..c2513bf 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -4065,6 +4065,7 @@ hid_lesstif_init ()
   lesstif_hid.flush_debug_draw        = lesstif_flush_debug_draw;
   lesstif_hid.finish_debug_draw       = lesstif_finish_debug_draw;
 
+  apply_default_hid (&lesstif_hid, 0);
   hid_register_hid (&lesstif_hid);
 #include "lesstif_lists.h"
 }
